include/SDL3_image/SDL_image.h

```diff
@@ -2519,7 +2531,21 @@ extern SDL_DECLSPEC IMG_AnimationEncoder * SDLCALL IMG_CreateAnimationEncoderWit
  * \sa IMG_CreateAnimationEncoderWithProperties
  * \sa IMG_CloseAnimationEncoder
  */
-extern SDL_DECLSPEC bool SDLCALL IMG_AddAnimationEncoderFrame(IMG_AnimationEncoder *encoder, SDL_Surface *surface, Uint64 duration);
+extern SDL_DECLSPEC bool SDLCALL IMG_AddAnimationEncoderFrame(IMG_AnimationEncoder *encoder, SDL_Surface *surface, Uint64 delay);
+
+
+/**
+* Get the encoder status indicating the current state of the encoder.
+*
+* \param encoder the encoder to get the status of.
+* \returns the status of the underlying encoder, or IMG_CODER_STATUS_INVALID if the given
+*          encoder is invalid.
+*
+* \since This function is available since SDL_image 3.4.0.
+*
+* \sa IMG_AddAnimationEncoderFrame
+*/
+extern SDL_DECLSPEC IMG_CoderStatus SDLCALL IMG_GetAnimationEncoderStatus(IMG_AnimationEncoder *encoder);
```

Comment:
We don't need this function for symmetry, we only need to disambiguate between error and EOF in the decode case.